maps/tether/tether-05-station1.dmm

```diff
@@ -16384,11 +16375,6 @@
 /obj/machinery/camera/network/civilian,
 /turf/simulated/floor/tiled,
 /area/tether/station/visitorhallway/lounge)
-"aCm" = (
-/obj/machinery/washing_machine,
-/obj/machinery/camera/network/civilian,
-/turf/simulated/floor/tiled,
-/area/tether/station/visitorhallway/laundry)
```

Comment:
Apparently this removes a washing machine? Or am I misreading this filediff?

Reply:
There are still three more in there and honestly, when ever more than two are used at once? I don't think that's too big a problem

Reply:
A'ight.